includes/amp-helper-functions.php

```diff
@@ -424,22 +424,23 @@ function amp_get_content_embed_handlers( $post = null ) {
 	 */
 	return apply_filters( 'amp_content_embed_handlers',
 		array(
-			'AMP_Twitter_Embed_Handler'     => array(),
-			'AMP_YouTube_Embed_Handler'     => array(),
-			'AMP_DailyMotion_Embed_Handler' => array(),
-			'AMP_Vimeo_Embed_Handler'       => array(),
-			'AMP_SoundCloud_Embed_Handler'  => array(),
-			'AMP_Instagram_Embed_Handler'   => array(),
-			'AMP_Issuu_Embed_Handler'       => array(),
-			'AMP_Meetup_Embed_Handler'      => array(),
-			'AMP_Vine_Embed_Handler'        => array(),
-			'AMP_Facebook_Embed_Handler'    => array(),
-			'AMP_Pinterest_Embed_Handler'   => array(),
-			'AMP_Playlist_Embed_Handler'    => array(),
-			'AMP_Reddit_Embed_Handler'      => array(),
-			'AMP_Tumblr_Embed_Handler'      => array(),
-			'AMP_Gallery_Embed_Handler'     => array(),
-			'WPCOM_AMP_Polldaddy_Embed'     => array(),
+			'AMP_Gutenberg_Categories_Handler' => array(),
```

Comment:
Since “Gutenberg“ project name will not be carried over when merged into core, we should just call it `AMP_Categories_Block_Handler`.